reactioncommerce/reaction
# Fix: Dashboard panel keeps re-opening during checkout

Resolves #1953, which was a regression reported from fixing #1932.

Issue: After making ActionView's (Sidebar) current display adjust based on the context in main checkout view, there needs to be checks to ensure that that behavior only happens the settings have not been configured.

**How to test:**
This update still keeps the behavior added in #1932 (so, that should be confirmed), but then ensure that:
- The sidebar comes out only if the expected settings is not yet configured

Note: This PR also refactored work from #1932 to leverage existing method calls instead of reinventing.
